packages/npm-dependencies-validation/test/index.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { getDependencyIssues } from "../src/index";
+import { expect, assert } from "chai";
+import * as fsPromises from "fs/promises";
+import { SinonMock, createSandbox } from "sinon";
+
+describe("index unit test", () => {
+  it("2 dependencies and 1 devDependency declared but are not installed", async () => {
+    const result = await getDependencyIssues({
+      fsPath: "./test/projects/no_deps_installed/package.json",
+    });
+    expect(result).to.have.lengthOf(3);
```

Comment:
evaluate use of https://www.chaijs.com/plugins/deep-equal-in-any-order/

Not sure its optimal, may be less verbose than the current implementation, however also harder to debug?

Reply:
I am not sure it is necessary for current tests.
Let's talk about it